packages/providers/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { exponentialBackoff } from './exponential-backoff';
+export { JsonRpcProvider } from './json-rpc-provider';
+export { Provider } from './provider';
+export { fetchJson } from './fetch_json';
```

Comment:
I refrained from renaming files to minimize changes. I typically use snake case so that was what I used for new files in the absence of convention. Definitely agree that this should be cleaned up and enforced to some extent.